go/libraries/doltcore/sqle/integration_tests/stockmarket_test.go

```diff
@@ -20161,6 +20167,40 @@ func TestInserts(t *testing.T) {
 	assert.Equal(t, uint64(6879), rowData.Len())
 }
 
+func TestInsertsWithIndexes(t *testing.T) {
+	dEnv := dtestutils.CreateTestEnv()
+	ctx := context.Background()
+
+	root, _ := dEnv.WorkingRoot(ctx)
+	var err error
+	root, err = sqle.ExecuteSql(dEnv, root, createTables)
+	require.NoError(t, err)
+
+	root, err = sqle.ExecuteSql(dEnv, root, createIndexes)
+	require.NoError(t, err)
+
+	root, err = sqle.ExecuteSql(dEnv, root, insertRows)
+	require.NoError(t, err)
+
+	table, _, err := root.GetTable(ctx, "daily_summary")
+	assert.NoError(t, err)
+	rowData, err := table.GetIndexRowData(ctx, "idx_country")
+	assert.NoError(t, err)
```

Comment:
You should always require.NoError if the next line is going to cause a null pointer panic otherwise

Reply:
That's a fantastic point, never even crossed my mind.